pkg/sql/logictest/testdata/logic_test/impure

```diff
@@ -33,6 +33,26 @@ SELECT count(*) FROM kab WHERE a=b
 ----
 0
 
+statement ok
+CREATE TABLE kabc (k INT PRIMARY KEY, a UUID, b UUID)
+
+statement ok
+INSERT INTO kabc VALUES (1, uuid_generate_v4(), uuid_generate_v4())
+
+statement ok
+INSERT INTO kabc VALUES (2, uuid_generate_v4(), uuid_generate_v4())
+
+statement ok
+INSERT INTO kabc VALUES (3, uuid_generate_v4(), uuid_generate_v4()),
+                       (4, uuid_generate_v4(), uuid_generate_v4()),
+                       (5, uuid_generate_v4(), uuid_generate_v4()),
+                       (6, uuid_generate_v4(), uuid_generate_v4())
+
+query I
+SELECT count(*) FROM kabc WHERE a=b
+----
+0
+
 statement ok
```

Comment:
i think just inserting `uuid_generate_v4()` rows into this table is fine (maybe even add `e UUID DEFAULT uuid_generate_v4()`).

Reply:
@otan thanks for the comment! I just updated the PR based on your comment and added calls to `uuid_generate_v4()` and `UUID DEFAULT`.